docs/pages/server-access/getting-started.mdx

```diff
@@ -0,0 +1,384 @@
+
+   ![Both Nodes in](../../img/server-access/teleport_ui.png)
+
+## Step 2a. Use tsh to connect to the cluster
+
+Now, we've got our cluster up and running. Let's see how easy it is to connect to our two nodes:
+
+1. On your local machine, log in through `tsh`: 
+
+   i. `tsh login --proxy=auth.example.com:443 --user=auth-admin`  
+   ii. You'll be prompted to supply the password and One Time Passcode we set up previously.
+
+2. `auth-admin` will now see:
+
+   ```bash
+   Profile URL:        https://auth.example.com:443
+     Logged in as:       auth-admin
+     Cluster:            auth.example.com
+     Roles:              access, admin, editor
+     Logins:             root, ubuntu, ec2-user
+     Kubernetes:         disabled
+     Valid until:        2021-04-30 06:39:13 -0500 CDT [valid for 12h0m0s]
+     Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
+   ```
+
+  - `auth-admin` is now logged into the `auth.example.com` cluster and `auth` Node via Teleport SSH - no third-party tool required.
+
+3. `auth-admin` can now execute the following to find the cluster's `nodenames` which are used for direct SSH connections:
+
+   i. `tctl nodes ls`  
+   ii. The `auth` node is located on the bottom line below:
+
+   ```bash
+   Nodename         UUID                                 Address        Labels                                
+   ---------------- ------------------------------------ -------------- ------------------------------------- 
+   ip-172-31-30-155 8f1e2719-3161-400f-b664-3279cf046511                env=example,hostname=ip-172-31-30-155 
+   ip-172-31-17-124 e7719967-b6eb-4fc0-8e53-00fac6d896d4 127.0.0.1:3022 env=example,hostname=ip-172-31-17-124 
+   ```
+
+4. Using the command `tsh ssh root@ip-172-31-17-124` locally, `auth-admin` can SSH into the `auth` Node and do the following:
+
+   i. Connect to other Nodes in the cluster by repeating the steps above.  
+   ii. Traverse the Linux file system.  
+   iii. Execute desired commands.  
+
+   - The `tsh ssh` command allows one to do anything they would if they were to SSH into a server using a third-party tool. 
+   - All commands executed by `auth-admin` are recorded and can be replayed in the Teleport Web interface.
+
+<video autoplay loop muted playsinline>
+  <source src="https://goteleport.com/teleport/videos/server-access/replay.mp4" type="video/mp4"/>
+  <source src="https://goteleport.com/teleport/videos/server-access/replay.webm" type="video/webm"/>
+Your browser does not support the video tag.
+</video>
+
+## Step 2b. Use tsh to introspect cluster
+
+1. Now, `auth-admin` has the ability to SSH into other Nodes within the cluster, traverse the Linux file system, and execute the command.
+
+   - They have visibility into all resources within the cluster due to their defined and assigned role.
+   - They can also quickly view any Node or grouping of Nodes that've been assigned a `label`.
+
+2. Executing `tctl nodes ls` displays:
```

Comment:
I would use `tsh ls` here - it's an equivalent command that is available in `tsh`